src/app/modules/item/pages/item-current-situation/item-current-situation.component.ts

```diff
@@ -1,25 +1,49 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnChanges, ViewChild } from '@angular/core';
 import { ItemData } from '../../services/item-datasource.service';
+import { Group } from '../../../group/http-services/get-group-by-id.service';
+import { RouterLinkActive } from '@angular/router';
+import { ItemType } from 'src/app/shared/helpers/item-type';
 
 @Component({
   selector: 'alg-item-current-situation',
   templateUrl: './item-current-situation.component.html',
   styleUrls: [ './item-current-situation.component.scss' ]
 })
-export class ItemCurrentSituationComponent {
-
+export class ItemCurrentSituationComponent implements OnChanges {
   @Input() itemData?: ItemData;
+  @Input() watchedGroup?: Group;
 
-  viewItems = [
-    { label: $localize`Log view`, value: 'log' },
-    { label: $localize`Chapter view`, value: 'chapter' },
-  ];
+  @ViewChild('historyTab') historyTab?: RouterLinkActive;
+  @ViewChild('chapterTab') chapterTab?: RouterLinkActive;
+  @ViewChild('chapterUserProgressTab') chapterUserProgressTab?: RouterLinkActive;
 
-  viewSelected = 0;
+  hideSelection = false;
+  showChapterUserProgress = false;
```

Comment:
Actually, these two variables aren't independent, the second one does not make sense is the first one is "true" from what I understand. 
It is also not straighforward while reading the code why "!showChapterUserProgress" makes you display the group progress tab.

So if I understand well what you want to do, you should have a variable with 3 states: no selector, selectors with  user chapter progress, selectors with group chapter progress. 
So just do, for instance:
```ts
selectors = 'none'|'withUserProgress'|'withGroupProgress';
```

Reply:
Fixed